### ISSUE TITLE
libjpeg-turbo - reporting potential security vulnerabilities

### ISSUE THREAD

**Issue by inferno-chromium** [libjpeg-turbo/libjpeg-turbo#134]
@dcommander

Our OSS-Fuzz fuzzing effort (https://testing.googleblog.com/2016/12/announcing-oss-fuzz-continuous-fuzzing.html) has located multiple security vulnerabilities in libjpeg-turbo using the fuzz target we developed.
https://clusterfuzz-external.appspot.com/v2/testcases?job=libfuzzer_asan_libjpeg-turbo&open=yes

These issue are now filed in security-protected monorail tracker and we'd like to find developers to take a look at them.

We will CC developers on these issues to give them access to stack traces and reproducer data. For that we'd need an e-mail with associated google account. We will also set up the process to auto-CC these people when we find more issues.

Check out https://github.com/google/oss-fuzz/blob/master/projects/libjpeg-turbo/project.yaml and we can get your emails setup for easy access to those current testcases.
(@kcc, @mikea as fyi)

**Comment by inferno-chromium** [comment]
oops, duplicate of #127. @dcommander, others - we are still waiting on your response.
